sys/hashes/cmac.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_hashes
+ * @{
+ *
+ * @file
+ * @brief       AES_CMAC implementation
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ *
+ * @}
+ */
+
+
+#include <stdio.h>
+#include <inttypes.h>
+#include <string.h>
+
+#include "hashes/cmac.h"
+#include "crypto/ciphers.h"
+
+#define MIN(a, b) a < b ? a : b
+
+void _xor128(uint8_t *x, uint8_t *y)
```

Comment:
Done. There aren't changes in codesize between void and static inline void in this case.
But there are with the old macros.

Cheers!